### ISSUE TITLE
The v14DetectionLoss loss function is problematic

### ISSUE THREAD

**Issue by xwgit2023** [zhangcbb/yolov14#7]
Hello, thank you for your outstanding work in open-sourcing YOLOv14.
I have a question regarding the v14DetectionLoss function, which appears to be inconsistent with the description in the paper.
In Section 3.7 of the YOLOv14 paper, the total loss is formulated as Ltotal = Ldet + λ1Ladv + λ2Lcross. However, this loss formulation is not implemented in the current codebase.
At line 395 of the tasks.py file, the model still uses the standard v8DetectionLoss function. The relevant code snippet is as follows:
```
              def init_criterion(self):
                  """Initialize the loss criterion for the DetectionModel."""
                  return E2EDetectLoss(self) if getattr(self, "end2end", False) else v8DetectionLoss(self)
```

Even when switching to v14DetectionLoss, there remain issues: the core data flow for domain_logits, domains, features and view_ids is still missing, and the default execution still falls back to the standard loss function.

I would be grateful if you could provide some clarification on this issue.
